### PR TITLE
adds confidence charts

### PR DESCRIPTION
**Pull request by jhoetter**
use [submodule](https://github.com/code-kern-ai/refinery-submodule-model/pull/10) and [frontend](https://github.com/code-kern-ai/refinery-ui/pull/60) from this branch.

When you jump into the clickbait project, you should see a confidence distribution chart on the overview page

### CODE REVIEW

File: controller/project/manager.py [hunk 1]
```diff
@@ -106,6 +106,17 @@ def get_label_distribution(
     return project.get_label_distribution(project_id, labeling_task_id, slice_id)
 
 
+def get_confidence_distribution(
+    project_id: str,
+    labeling_task_id: str,
+    slice_id: str = None,
+    num_samples: int = None,
```
Comment (JWittmeyer) [inline controller/project/manager.py]: type optional

File: graphql_api/query/project.py [hunk 2]
```diff
@@ -145,6 +153,21 @@ def resolve_label_distribution(
         auth.check_project_access(info, project_id)
         return manager.get_label_distribution(project_id, labeling_task_id, slice_id)
 
+    def resolve_confidence_distribution(
+        self,
+        info,
+        project_id: str,
+        labeling_task_id: str = None,
+        slice_id: str = None,
+        num_samples: int = 100,
```
Comment (JWittmeyer) [inline graphql_api/query/project.py:161]: is this wanted to be only ever 100 samples with not provided? feels a bit low

File: graphql_api/types.py [hunk 3]
```diff
@@ -420,6 +423,12 @@ def resolve_inter_annotator_matrix(self, info):
 
         raise ValueError(f"Can't match task type {self.task_type}")
 
+    def resolve_confidence_distribution(self, info):
+        confidence_scores = project.get_confidence_distribution(
+            self.project_id, self.id, num_samples=100
```
Comment (JWittmeyer) [inline graphql_api/types.py:427]: 100 feels low as a sample imo

File: controller/project/manager.py (hunk 1 above)
Comment (jhoetter) [reply controller/project/manager.py]: done

File: graphql_api/query/project.py (hunk 2 above)
Comment (jhoetter) [reply graphql_api/query/project.py:161]: for now, 100 are ok :)

Comment (FelixKirsch) [inline graphql_api/query/project.py]:
```suggestion
        labeling_task_id: Optional[str] = None,
        slice_id: Optional[str] = None,
        num_samples: int = 100,
```
typing :), may need to add at the top: from typing import Optional

File: graphql_api/types.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+from importlib.metadata import distribution
```
Comment (FelixKirsch) [inline graphql_api/types.py]:
```suggestion
```
unused import

**Review by FelixKirsch — APPROVED** [review]

**Pull request merged**